src/main/java/com/uspray/uspray/domain/History.java

```diff
@@ -0,0 +1,63 @@
+package com.uspray.uspray.domain;
+
+import com.uspray.uspray.Enums.PrayType;
+import com.uspray.uspray.common.domain.AuditingTimeEntity;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class History extends AuditingTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "history_id")
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member;
+
+    private String content;
+
+    private Integer count;
+
+    private LocalDate deadline;
+
+    @Column(name = "origin_pray_id")
+    private Long originPrayId;
+
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    private PrayType prayType;
+
+    @NotNull
+    @ManyToOne
```

Comment:
ManyToOne 타입에 지연로딩 걸어주면 좋을 것 같습니다!
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```